### PR TITLE
Add flutter::Pipeline to //synchronization.

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

**Comment by chinmaygarde** [comment]
cc @abarth @jason-simmons

**Comment by jason-simmons** [comment]
LGTM

**Comment by chinmaygarde** [comment]
Thanks!

File: synchronization/pipeline.cc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/synchronization/pipeline.h"
+
+namespace flutter {
+
+//
```
Comment (abarth) [inline synchronization/pipeline.cc]: ?

Comment (abarth) [inline synchronization/pipeline.cc]: If this file just exists to check the build for the header, I usually skip the namespace and write a comment to that effect.

Comment (chinmaygarde) [inline synchronization/pipeline.cc]: Pretty much. Yeah. The comment is to make clang-format not collapse that to a single line :)

Comment (chinmaygarde) [inline synchronization/pipeline.cc]: Will updated in another patch.

**Pull request merged**